.github/workflows/test-sdks-remote.yml

```diff
@@ -4,9 +4,30 @@ on:
   push:
     branches:
       - main
+      - tp/workflows/round2
   workflow_dispatch:
 
 jobs:
+
+  test-java-sdk:
```

Comment:
lots of duplicated code, have you considered using a matrix?

Unchecked example from ChatGPT:
```yaml
test-sdks:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        sdk:
          - { repo: "java-server-sdk", workflow: "lint-test-sdk.yml", ref: "tp/workflows/remote" }
          - { repo: "php-sdk", workflow: "run-tests.yml", ref: "main" }
          - { repo: "js-client-sdk", workflow: "lint-test-sdk.yml", ref: "tp/workflows/remote" }
          - { repo: "python-sdk", workflow: "lint-test-sdk.yml", ref: "tp/workflows/remote" }
    steps:
    - name: Display workflow details
      shell: bash
      run: |
        echo "Testing eppo-exp/${{ matrix.sdk.repo }}"
    - name: Run remote workflow
      uses: convictional/trigger-workflow-and-wait@v1.6.1
      with:
        owner: Eppo-exp
        repo: ${{ matrix.sdk.repo }}
        workflow_file_name: ${{ matrix.sdk.workflow }}
        ref: ${{ matrix.sdk.ref }}
        github_token: ${{ secrets.AUTH_TOKEN }}
        wait_interval: 10
        propagate_failure: true
        trigger_workflow: true
        wait_workflow: true
```

Reply:
I hadn't, in fact. 🧑‍🍳